src/SqlPersistence/Config/SqlDialect_Oracle.cs

```diff
@@ -12,6 +13,8 @@ public abstract partial class SqlDialect
         /// </summary>
         public partial class Oracle : SqlDialect
         {
+            volatile PropertyInfo bindByNameProperty;
```

Comment:
It is not ideal but I think better than `.GetType().GetMethod()`. I put volatile here to ensure that eventually the value can be read by other threads. To be honest I don't think it is strictly necessary.